workos/__about__.py

```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "4.7.0"
+__version__ = "4.8.0"
```

Comment:
Could we do this in a separate PR for any other changes that may be included in the release?

Reply:
are you an owner for the python sdk? i do it in 1 so that it doesn't require 2 owners checks (been blocked by this before)
but i'll grab whatever changes that were merged before this when i cut a release

Reply:
Yeah I'm a _Pythonista_, so can definitely approve the bump whenever. I suppose it's nice to have the clear history of when the version was bumped/what it included.

![image](https://github.com/workos/workos-python/assets/9063128/def8cab3-58ef-4919-8bef-41243c182ecb)

Reply:
great, just undone